frontend/src/pages/ErrorsV2/ErrorInstance/AiErrorSuggestion.tsx

```diff
@@ -205,6 +218,19 @@ export const AiErrorSuggestion = ({ errorObjectId }: Props) => {
 					>
 						Refresh Suggestion
 					</Button>
+					<Button
```

Comment:
It does in testing, but only for a split second. Checked the loading, networkStatus states and they match, it only loads for like 1 second. I'm assuming this is cause of caching though, since there's no way the API is returning so quick. It looks like it returns data from cache (triggering onComplete, so we can't set loading state through that either) and then that cached data is updated like 5 seconds later. 

Tried updating fetchPolicy to no-cache or network-only, but doesn't seem to be doing anything. Could be something to investigate in another PR.

Reply:
Opened an issue here to track it: https://github.com/highlight/highlight/issues/5947